p2p/host/autonat/grafana-dashboards/autonat.json

```diff
@@ -0,0 +1,658 @@
+                                                                }
+                                                        }
+                                                ]
+                                        },
+                                        {
+                                                "matcher": {
+                                                        "id": "byName",
+                                                        "options": "OK"
+                                                },
+                                                "properties": [
+                                                        {
+                                                                "id": "color",
+                                                                "value": {
+                                                                        "fixedColor": "green",
+                                                                        "mode": "fixed"
+                                                                }
+                                                        }
+                                                ]
+                                        }
+                                ]
+                        },
+                        "gridPos": {
+                                "h": 8,
+                                "w": 12,
+                                "x": 12,
+                                "y": 15
+                        },
+                        "id": 6,
+                        "options": {
+                                "legend": {
+                                        "calcs": [],
+                                        "displayMode": "list",
+                                        "placement": "bottom",
+                                        "showLegend": true
+                                },
+                                "tooltip": {
+                                        "mode": "single",
+                                        "sort": "none"
+                                }
+                        },
+                        "targets": [
+                                {
+                                        "datasource": {
+                                                "type": "prometheus",
+                                                "uid": "${DS_PROMETHEUS}"
+                                        },
+                                        "editorMode": "code",
+                                        "expr": "increase(libp2p_autonat_server_dialresponse_total[$__rate_interval])",
+                                        "legendFormat": "{{response_status}}",
+                                        "range": true,
+                                        "refId": "A"
+                                }
+                        ],
+                        "title": "server: DialResponse By Type",
```

Comment:
I have to admit, I'm always confused who's the client and server when thinking about AutoNAT. Maybe "Outgoing Dials" would be harder to mix up?

Reply:
outgoing does seem better. will fix.

Reply:
this is fixed.